examples/next/docs/js/demo/src/constants.js

```diff
@@ -1201,5 +1201,33 @@ export const data = [
   ]
 ];
 
+const randomName = () =>
+  ["عمر", "علي", "عبد الله", "معتصم"][Math.floor(Math.random() * 3)];
+const randomCountry = () =>
+  ["تركيا", "مصر", "لبنان", "العراق"][Math.floor(Math.random() * 3)];
+const randomDate = () =>
+  new Date(Math.floor(Math.random() * Date.now())).toLocaleDateString()
```

Comment:
`toLocaleDateString()` uses our browser's locale. Shouldn't this be `toLocaleDateString('ar')` or so?

Reply:
Okay, this will hurt :) https://stackoverflow.com/questions/40859180/how-to-format-date-into-arabic-with-english-numerals-for-date-and-year-in-javasc